docs: add information about missing opampbridge crd (#3489)
SumoLogic/sumologic-documentation: docs/send-data/kubernetes/v4/how-to-upgrade.md

            sourceLabels: [__name__]
```

#### Apply missing Custom Resource Definition for OpenTelemetry Operator

> **When??** If you already uses OpenTelemetry Operator, and it is in version before `0.42.0`
> 
> Please apply the following command on your cluster:

Reviewer: ```suggestion
**When??** If you already use OpenTelemetry Operator in a version below `0.42.0`, apply the following command on your cluster:
```
